### DISCUSSION TITLE
Is it possible to use wildcards or ranges of hosts or IP address?

### DISCUSSION THREAD

**Discussion by jringoot** [ParallelSSH/parallel-ssh#294] [Q&A] [answered]
In parallel you can introduce a hostrange with {n..N} in the name or ip address
In ansible one can introduce ranges with  [n:N] in the name or ip address
Is this possible in parallel-ssh? What is the syntax?

**Comment by pkittenis** [comment, marked as answer]
Thanks for the interest.

No. Generate the host list in python.

```
hosts = [f"192.168.1.{x}" for x in range(1,10)]
client = ParallelSSHClient(hosts)
```

**Reply by jringoot** [reply]
Thanks for your quick response, apparently backticks are accepted, which is also cool, see screenshot.
But I dont get the output

![Screenshot from 2021-04-13 21-31-34](https://user-images.githubusercontent.com/2686933/114609952-abbdae00-9c9f-11eb-9b74-f55cef895e23.png)

**Reply by pkittenis** [reply]
No idea what's been shown there. `parallel-ssh` cmd line script does not come from this project.

If you have a question or a bug, please make a new question or raise an issue that shows some python code using this library.

There are no restrictions in what the cmd can be in the library.
